js/parts-more/Polar.js

```diff
@@ -529,8 +514,16 @@ if (seriesTypes.column) {
                             }
                         }
                     }
+                    else {
+                        point.shapeType = 'arc';
```

Comment:
in IF and ELSE there is 'arc'.

Consider to set the `arc` by default and add 
```
if (point.graphic.element.nodeName === 'path') {
   point.shapeType = 'path';
}
```
only in IF

Reply:
Corrected, thanks for the suggestion!